app/controllers/items_controller.rb

```diff
@@ -5,7 +5,7 @@ class ItemsController < ApplicationController
   def has_user
     unless current_user
       flash[:warning] = 'You must be logged in to perform this action.'
-      redirect_to new_user_path
+      redirect_to items_path
```

Comment:
why was this changed?

Reply:
i don't think this should be changed

Reply:
New user path doesnt exist anymore @amycsong98 , since we can't create a user its just sign in with google SSO

Reply:
cool